test/Directory.Build.props

```diff
@@ -1,17 +1,13 @@
 <Project>
```

Comment:
👍 Love the `Directory.Build.props`, hopefully it'll be used more broadly after net6 all up.